static/js/stream_color.js

```diff
@@ -128,7 +128,7 @@ exports.sidebar_popover_colorpicker_options_full = {
     showInput: true,
     flat: true,
     cancelText: "",
-    chooseText: "choose",
+    chooseText: i18n.t("choose"),
```

Comment:
The thing is that I can't even reach `ensure_i18n`, because `i18n` isn't defined here, for some reason.

I've seen that the `i18n` variable is declared in `static/js/src/main.js`, but couldn't find what defines the order of execution of all JS files (just to see if `stream_color.js` is running before `main.js`).

Could you please offer some guidance?